src/components/review/PSAMetaData.js

```diff
@@ -90,9 +91,13 @@ export default class PSAMetaData extends React.Component<Props, State> {
     let creator = '';
     let dateEdited;
     let editor = '';
+    const rcmRiskFactors = psaNeighbors.get(RCM_RISK_FACTORS, Map());
     const { [DATE_TIME]: psaCreationDate } = getEntityProperties(scores, [DATE_TIME]);
+    const { [CONTEXT]: caseContext } = getEntityProperties(rcmRiskFactors, [CONTEXT]);
     dateCreated = DateTime.fromISO(psaCreationDate);
 
+    const trimmedCaseContext = caseContext.split(' ')[0];
```

Comment:
This will return an empty string if caseContext leads with a space

' something'.split(' ')[0]
// -> ""

Reply:
this will never happen, but I can change it. We set this property, it never something that they ever manually enter. It's been the same string constant since the app has existed. 'Court (<county>)' or 'Booking'.

Reply:
I see. A simple caseContext.trim().split(' ')[0] would take care of this.